docs/flash_drive_instructions.rst

```diff
@@ -0,0 +1,20 @@
+.. instructions for using a USB that can install pytentiostat
+
+=============================================================================
+Instructions for installing pytentiostat onto a *Windows* computer with a USB
+=============================================================================
+
+**You must have a USB drive with pytentiostat and pytent_env to use this method.**
+**If you do not have such a drive, you can download the source code at** https://github.com/juami/pytentiostat
```

Comment:
These instructions are confusing I think.  Much better is to give people step by step instructions, and then alternatives if things fail.

For example, I am already stumped on the first one.   I have a USB drive (though I don't really need to be told I need this if I am installing from a USB), but I have no idea if I have pytent_env, and no idea how to check or what to do if I don't have it.  => frustration and dissatisfaction.

Can it be written more like (without the bold warnings at the top of what can go wrong).
 1. start and log in to your computer
 2. place the USB drive into a USB slot in your computer
 3. <describe what they will see, then what to do, "double click on the folder icon" or sthg like that>
 4. Drag and drop the folder named `pytentiostat` onto your desktop (or into the folder where you want it located)
 5. Look inside the `pytentiostat` folder for `pytent_env` and inside that folder for a file called \
  `pytent_GUI.bat`. 
 6. To run the graphical user interface, double click on `pytent_GUI.bat` to run the software.
 7. If you prefer to run the command-line interface to the software (less common) , double click on `pytent_cmd.bat`